src/builders/util.js

```diff
@@ -0,0 +1,5 @@
+export function findNearestElement(view, selector) {
+  if (!view || view.matches("html")) return null;
+  if (!selector) return null;
+  return document.querySelector(selector) || findNearestElement(view.parentElement, selector);
```

Comment:
Disregard previous comments as they were based off the intent rather than the implementation. I think you're right.